docs/user/troubleshooting.md

```diff
@@ -263,3 +494,7 @@ If appropriate, file a [network operator](https://github.com/openshift/cluster-n
 [aws-key-pairs]: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html
 [kubernetes-debug]: https://kubernetes.io/docs/tasks/debug-application-cluster/
 [machine-config-daemon-ssh-keys]: https://github.com/openshift/machine-config-operator/blob/master/docs/Update-SSHKeys.md
+[cluster-version-operator]: https://github.com/openshift/cluster-version-operator/blob/master/README.md
```

Comment:
nit: I usually keep these references alphabetized, but that's not worth holding up this commit